src/sentry/integrations/pagerduty/client.py

```diff
@@ -31,7 +31,7 @@ def request(self, method, path, headers=None, data=None, params=None):
     def send_trigger(self, data):
         # expected payload: https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2
         # for now, only construct the payload if data is an event
-        if isinstance(data, EventCommon):
+        if isinstance(data, Event):
```

Comment:
This is fine because only eventstore.models.Event and models.Event extend EventCommon. Though Do we need to merge PR 16558 first ?

Reply:
It's not dependent on that, we're already no longer using that model anywhere